README.md

```
@@ -21,6 +21,51 @@
   - Looping (`for`, `while`, `until`)
 - Functions in Shell Scripts  
 - Exit Codes and Status (`$?`, `exit` command)
+- Basic codes(Executing your first shell scripts)
+- Writing Your First Shell Script (.sh file, shebang #!/bin/bash)
+
+#!/bin/bash
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)